### DISCUSSION TITLE
Better UX for lattice errors

### DISCUSSION THREAD

**Discussion by scottwn** [AgnostiqHQ/covalent#918] [Ideas]
When there is an error in a lattice, the trace is rendered as a long variable width string with no line breaks in the sidebar. There should be some way to get the trace in a more readable format, either directly in the sidebar, or in a popout or detail window, or something that could be retrieved programmatically by the user. The long string is ugly and not very helpful.

**Comment by scottwn** [comment]
example of the ugly error string
![Screen Shot 2022-07-25 at 6 20 46 PM](https://user-images.githubusercontent.com/25555884/180883657-60752191-eb49-4393-ad4e-e66869111b45.png)
this error was generated by the workflow in https://github.com/AgnostiqHQ/covalent/issues/917

**Reply by Andrew-S-Rosen** [reply]
I believe this is also true for the "Description" in the Overview section. For instance, I have an electron that looks like

```python3
@ct.electron
def StaticJob(...):
    """
    Function to carry out a static calculation.

    Parameters
    ----------
    atoms
        .Atoms object
    asap_cutoff
        If an ASAP-style cutoff should be used.

    Returns
    -------
    Dict
        Summary of the run.
    """
```

which is transformed to the following in the UI.

<img width="331" alt="image" src="https://user-images.githubusercontent.com/8674072/233552380-d7e7953f-f6e8-4ab3-b6e6-c7a9619d2a36.png">
